companion/src/mdichild.cpp

```diff
@@ -87,6 +89,22 @@ void MdiChild::refresh(bool expand)
   updateTitle();
 }
 
+void MdiChild::onItemSelected(QModelIndex idx)
+{
+  emit copyAvailable(isModel(idx) ? true : false);
+}
+
+bool MdiChild::isModel(QModelIndex idx)
+{
+  return (modelsListModel->getModelIndex(idx) >= 0 ? true : false);
+}
+
+bool MdiChild::isCategory(QModelIndex idx)
+{
+  if (isModel(idx)) return false;
+  else return (modelsListModel->getCategoryIndex(idx) >= 0 ? true : false);
```

Comment:
You could actually just `return (modelsListModel->getCategoryIndex(idx) >= 0);`  here if you'd like.